.github/workflows/emacs.yml

```diff
@@ -14,4 +14,4 @@ jobs:
       uses: actions/checkout@v1
 
     - name: Test
-      run: cd test; emacs --batch -q -l ert -l ../go-mode.el -l go-indentation-test.el -f ert-run-tests-batch-and-exit
+      run: cd test; emacs --batch -q -l ert -l ../go-mode.el $(for t in `ls *-test.el`; do echo -n "-l $t "; done) -f ert-run-tests-batch-and-exit
```

Comment:
You don't need to invoke for this `ls`: `for t in *-test.el`.

Reply:
Done